invenio_vocabularies/contrib/awards/jsonschemas/awards/award-v1.0.0.json

```diff
@@ -42,6 +42,48 @@
     },
     "program": {
       "type": "string"
+    },
+    "subjects": {
```

Comment:
minor: we can [copy the structure](https://github.com/slint/invenio-rdm-records/blob/26448945f73c56d5ca2724ec9970409b5e5ab6b9/invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json#L374-L390) we have for records on subjects, i.e.:

```python
{
    "subjects": [
        {"id": "euroscivoc:1234"},
        {"subject": "free form subject"},
    ]
}
```